optimade/models/entries.py

```diff
@@ -147,12 +177,48 @@ class EntryResource(Resource):
 Database-provider-specific properties need to include the database-provider-specific prefix (see section on Database-Provider-Specific Namespace Prefixes).""",
     )
 
+    meta: Optional[EntryMetadata] = StrictField(
+        None,
+        description="""A dictionary, containing entry and property-specific metadata for a given entry.""",
```

Comment:
Again updating wrt. the spec 

```suggestion
        description="""A [JSON API meta object](https://jsonapi.org/format/1.1/#document-meta) that is used to communicate metadata. 
See [JSON Response Schema: Common Fields](https://github.com/Materials-Consortia/OPTIMADE/blob/develop/optimade.rst#json-response-schema-common-fields) for more information about this field.""",
```

Reply:
The link in line 182 points to the top level meta field. This meta describes per entry meta. So I'll only add line 181